File: FlipLearn/back/back/Core/Domain/Models/Card.cs

namespace back.Core.Domain.Models
{
    public class Card
    {
        public long Id { get; set; }
        public int DictionaryId { get; set; }
        public string Term { get; set; }
        public string? Meaning { get; set; }
        public string? Translation { get; set; }
        public string Status { get; set; }
    }
}

File: FlipLearn/mobile/app/src/main/java/com/example/fliplearn_final/domain/model/Card.kt
package com.example.fliplearn_final.domain.model

import java.time.Instant

data class Card(
    val cardId: Int = 0,
    val dictionaryId: Int = 0,
    val term: String,
    val meaning: String,
    val translation: String,
    val createdAt: Instant = Instant.now()
)


